apps/web/src/common/composables/amcharts5/index.ts

```diff
@@ -152,6 +152,10 @@ export const useAmcharts5 = (
             if (!state.root) throw new Error('No root');
             return createBullet(state.root as Root, settings);
         },
+        createLabel: (settings: am5.ILabelSettings): am5.Label => {
+            if (!state.root) throw new Error('No root');
```

Comment:
Question: How about we put this part into a helper function? I thought it would be beneficial to make it shorter and simpler where it is used. I'm curious if there are any other opinions!

Reply:
Thank you for opinion! I will refer to it during refactoring.